bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,37 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    I18N_METRICS_NAMESPACE = 'I18n'.freeze
+
+    def self.report_runtime(addtl_dimensions)
```

Comment:
Please use `addtl_dimensions = []` here and in `.log_metric` for cases when we don't need to provide `addtl_dimensions`

Reply:
I would also recommend using `*addtl_dimensions` instead `addtl_dimensions = []`. In this case, we must provide it comma-separated `report_runtime({a: 1}, {b: 2})` instead of as an array `report_runtime([{a: 1}, {b: 2}])`